exports/celo.json

```diff
@@ -1085,6 +1085,751 @@
         }
       ]
     },
+    "LBPManagerFactory": {
```

Comment:
I expected to see `LBPManagerFactoryNoAccessControl`

Reply:
Mistake on my side, now exported correctly

Reply:
Maybe you forgot to push, but I did not see any change in the Repo